Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Ammunition/Box/antimateriel.yml

```
@@ -0,0 +1,29 @@
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+    - state: rubber
+
+- type: entity
+  parent: BaseMagazineBoxAntiMateriel
+  id: LPPMagazineBoxAntiMaterielCocnterePiercing
+  name: ammunition box (.60 Cocntere-Piercing)
+  components:
+  - type: BallisticAmmoProvider
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)